internal/internal_public.go

```diff
@@ -58,7 +58,7 @@ type (
 		// - RespondActivityTaskCompletedRequest
 		// - RespondActivityTaskFailedRequest
 		// - RespondActivityTaskCancelRequest
-		Execute(task *s.PollForActivityTaskResponse) (interface{}, error)
+		Execute(taskList string, task *s.PollForActivityTaskResponse) (interface{}, error)
```

Comment:
We can make tasklist also part of PollForActivityTaskResponse. All other params are in there, tasklist seems relevant as well.

Reply:
I think it is overkill as poller knows which task list it polls on.

Reply:
Makes sense.